pkg/utils/apiresources.go

```diff
@@ -180,17 +198,22 @@ func (r *DisabledResourceConfig) IsResourceDisabled(gvk schema.GroupVersionKind)
 	return false
 }
 
-// DisableGroup to disable group.
-func (r *DisabledResourceConfig) DisableGroup(g string) {
+// AddGroup to store group in the resource config.
```

Comment:
```suggestion
// AddGroup stores group in the resource config.
```

Could you please help us to fix the comments? The existing codes are not following the golang best practices.  :(

` Comments documenting declarations should be full sentences, even if that seems a little redundant. This approach makes them format well when extracted into godoc documentation. Comments should begin with the name of the thing being described and end in a period:`
https://go.dev/wiki/CodeReviewComments#comment-sentences